### ISSUE TITLE
Bug (possibly in `reformMap`?) causes dates from `ParameterEditor` to be off by a year

### ISSUE THREAD

**Issue by anth-volk** [PolicyEngine/policyengine-app#1534]
For example, in the below image: the reform is set as starting `2022-01-01` (confirmed via the `reformData` variable when running locally), but the chart shows it starting `2021-01-01`. 

<img width="1440" alt="Screen Shot 2024-04-01 at 11 32 11 PM" src="https://github.com/PolicyEngine/policyengine-app/assets/14987227/c4f01fab-1135-4fdb-b7b8-4f6fdd724cd5">

**Comment by abhcs** [comment]
I don't think this is a bug. The reforms are not shown on the chart. Instead, the reformed parameter (in canonical/minimal form) is shown. The explanation in https://github.com/PolicyEngine/policyengine-app/issues/1520#issuecomment-2030739780 may help.
